lib/Signaling/Manager.php

```diff
@@ -36,6 +36,7 @@ public function isCompatibleSignalingServer(IResponse $response): bool {
 		$features = explode(',', $featureHeader);
 		$features = array_map('trim', $features);
 		return in_array('audio-video-permissions', $features, true)
+			&& in_array('federation', $features, true)
```

Comment:
could go into `getSignalingServerMissingFeatures` Or we make it a hard requirement?

Reply:
I set it as a hard requirement for simplicity, as if it is mandatory two servers with v20 and external signaling mode would be compatible without having to check the features of both signaling servers. But given that compatibility between servers still needs to be properly implemented fine by me to make it optional rather than mandatory.

Reply:
Okay lets keep it for now